backend/src/packages/auth/auth.service.ts

```diff
@@ -22,9 +22,14 @@ class AuthService {
 
     const user = await this.userService.findByEmail(email);
 
-    if (!user) {
-      throw new UserNotFoundError();
```

Comment:
Turn back user exists assertion

Reply:
If user not found, the error should be shown before password comparison.